re-enable pkg/skaffold/schema/validation/samples_test.go

##Description

integration some tests in pkg/skaffold/schema/validation/samples_test.go were temporarily disabled due to v2 migration without completing v2 render
Some disabled tests may fail when enabling it due to test skaffold.yaml don't have render config setup in skaffold.yaml, adding that part should fix the issue.
now v2 render is complete, as the goal of v2.0 alpha 1 milestone, we should make all integration tests pass
##ASK

re-enable all tests in pkg/skaffold/schema/validation/samples_test.go

Currently failing with:
```
    --- FAIL: TestParseSamples/ko-platforms.yaml (0.00s)
        /usr/local/google/home/aprindle/skaffold/pkg/skaffold/schema/validation/samples_test.go:80: Checking ../../../../docs/content/en/samples/builders/ko-platforms.yaml...
        /usr/local/google/home/aprindle/skaffold/pkg/skaffold/schema/validation/samples_test.go:92: unexpected error: unable to parse config: yaml: unmarshal errors:
              line 6: field platforms not found in type latest.KoArtifact
```

--- a/docs-v2/content/en/samples/builders/ko-platforms.yaml
+++ b/docs-v2/content/en/samples/builders/ko-platforms.yaml
@@ -1,7 +1,5 @@
 build:
+  platforms: ["linux/amd64", "linux/arm64"]
   artifacts:
   - image: foo
     ko: {}
-    platforms:
-    - linux/amd64
-    - linux/arm64

--- a/docs/content/en/samples/builders/ko-platforms.yaml
+++ b/docs/content/en/samples/builders/ko-platforms.yaml
@@ -1,7 +1,5 @@
 build:
+  platforms: ["linux/amd64", "linux/arm64"]
   artifacts:
   - image: foo
-    ko:
-      platforms:
-      - linux/amd64
-      - linux/arm64
+    ko: {}

--- a/pkg/skaffold/schema/validation/samples_test.go
+++ b/pkg/skaffold/schema/validation/samples_test.go
@@ -47,9 +47,6 @@ var (
 // Test that every example can be parsed and produces a valid
 // Skaffold configuration.
 func TestParseExamples(t *testing.T) {
-	// TODO: add examples for v2
-	t.SkipNow()
-
 	parseConfigFiles(t, "../../../../examples")
 	parseConfigFiles(t, "../../../../integration/examples")
 	parseConfigFiles(t, "../../../../integration/testdata/regressions")
@@ -58,9 +55,6 @@ func TestParseExamples(t *testing.T) {
 // Samples are skaffold.yaml fragments that are used
 // in the documentation.
 func TestParseSamples(t *testing.T) {
-	// TODO: add sample for v2
-	t.SkipNow()
-
 	paths, err := walk.From(samplesRoot).WhenIsFile().CollectPaths()
 	if err != nil {
 		t.Fatalf("unable to list samples in %q", samplesRoot)
@@ -102,7 +96,6 @@ func checkSkaffoldConfig(t *testutil.T, yaml []byte) {
 	t.CheckNoError(err)
 }
 
-//nolint:golint,unused
 func parseConfigFiles(t *testing.T, root string) {
 	groupedPaths, err := walk.From(root).WhenHasName("skaffold.yaml").CollectPathsGrouped(1)
 	if err != nil {

